cpp/src/Ice/ThreadPool.cpp

```diff
@@ -614,8 +605,7 @@ IceInternal::ThreadPool::run(const EventHandlerThreadPtr& thread)
             //
             // Get the next ready handler.
             //
-            while (_nextHandler != _handlers.end() &&
-                   !(_nextHandler->second & ~_nextHandler->first->_disabled & _nextHandler->first->_registered))
```

Comment:
I find all these bitmask computations hard to read.

Now that _disabled is gone (== always 0), I just removed all `~..._disabled`.